service/patch.go

```diff
@@ -125,6 +125,11 @@ func (uis *UIServer) diffPage(w http.ResponseWriter, r *http.Request) {
 			http.StatusInternalServerError)
 		return
 	}
+	if fullPatch == nil {
```

Comment:
I think we need to add a check on L43 of this file as well

Reply:
Thank you!